.travis.yml

```diff
@@ -12,13 +12,12 @@ env:
   - SECRET_KEY="i1bn=oly)w*2yl-5yc&f!vvgt)p)fh3_2$r#spa!*sw36f5ov7"
   - GH_TOKEN="dummygithubtoken"
 
+# venv is already activated
 before_script:
-  - psql -c "CREATE DATABASE vulnerablecode;" -U postgres
-  - ./manage.py migrate
+  - make postgres ACTIVATE=
```

Comment:
What does `ACTIVATE=` mean here?

Reply:
This implies that the virtualenv is already activated i.e. the ACTIVATE command doesn't do anything (nop).
Commented in line 45.

Reply:
As per our [discussion](https://github.com/nexB/vulnerablecode/wiki/WeeklyMeetings#meeting-on-tuesday-2021-07-27-at-1300-utc), I'll remove travisCI.